static/app/components/events/aiAutofix/useAiAutofix.tsx

```diff
@@ -0,0 +1,70 @@
+import {useCallback, useEffect, useRef, useState} from 'react';
+
+import {EventMetadataWithAutofix} from 'sentry/components/events/aiAutofix/types';
+import {Group} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+const POLL_INTERVAL = 5000;
+
+export const useAiAutofix = (_group: Group) => {
+  const organization = useOrganization();
+  const api = useApi();
+  const {
+    data: group,
+    isError,
+    refetch: dataRefetch,
+    error,
+    dataUpdatedAt,
+  } = useApiQuery<Group>([`/organizations/${organization.slug}/issues/${_group.id}/`], {
```

Comment:
It would be nice if we could _just_ query the autofix data, but if not let's at least pass in some collapse params so we're getting less junk in the response. I think that would be `collapse: ['release', 'tags']`

Reply:
Good idea, made a separate get endpoint for just the autofix data instead